mod-audit-server/src/main/java/org/folio/services/impl/OrderAuditEventServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.services.impl;
+
+import io.vertx.core.Future;
+import io.vertx.pgclient.PgException;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.folio.dao.OrderEventDao;
+import org.folio.kafka.exception.DuplicateEventException;
+import org.folio.rest.jaxrs.model.OrderAuditEvent;
+import org.folio.services.OrderAuditEventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service("orderAuditEventService")
```

Comment:
We don't need to specify service name manually in this case, just @Service will be enought

Reply:
fixed